### ISSUE TITLE
Option to disable transliteration

### ISSUE THREAD

**Issue by shihong5858** [daniel-j/send2ereader#24]
First of all, thank you so much for providing such a great online tool to help Kobo users to transfer e-books in a convenient way.

One issue troubles me a lot by using the tool, is that when I transfer Chinese/Japanese books, their name would be translated to English form and it's really hard to read/find when I connect the device using USB.

I checked the code and found it's processed by the transliteration plugin, I assume it is to help to convert file names with proper encoding. Therefore I wonder if the web tool can provide an additional option to bypass this step so it would be super helpful to non-Latin language users.

Thank you.

**Comment by daniel-j** [comment]
Hello and thank you for your input. I think I added transliteration because Kobo or Kindle browser only can download files with some characters in filename. If filename would contain an unexpected glyph it would silently fail. Been a while since I added that feature, but I can take a closer look. I thought transliteration was better than just stripping the glyphs, in case the filename only contained kanji characters for example nothing but extension would remain.

**Comment by shihong5858** [comment]
Hmmm I see. From I just tested the Kobo seems able to accept non-Latin characters:

![tempImagetM9sqo](https://github.com/daniel-j/send2ereader/assets/3605219/37b8364e-b893-4b39-982c-e9495d06bff7)

I used a Notion link so there were gibberish UUID in the file now.
I do understand there are cases that filenames with glyphs would cause issues. However if possible, it would be greatly helpful for non-Latin language users to have an option (even its' not by default checked) to bypass the transliteration process.
